doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -137,6 +151,10 @@ Matter
         1. Copy the content of the image configuration file :file:`prj.conf` located in the `sysbuild/<image_name>` directory (for example,  :file:`sysbuild/mcuboot`) to the :file:`prj.conf` file located in the :file:`child_image/<image_name>` directory.
         #. Copy the content of the board configuration file located in the :file:`sysbuild/<image_name>/boards` directory (for example, :file:`sysbuild/mcuboot/boards/nrf52840dk_nrf52840.conf`) to the board file located in the :file:`child_image/<image_name>/boards` directory.
 
+      * All Partition Manager configuration files (:file:`pm_static` files) with the suffix ``release`` has been removed from all samples.
+        Those files are redundant now, since the new build system allows to use the file without the additional suffix if you use :makevar:`FILE_SUFFIX` if it is available in the projects directory.
```

Comment:
```suggestion
        Those files are now redundant, since the new build system allows using the file without the additional suffix if you use :makevar:`FILE_SUFFIX` and it is available in the project's directory.
```